Sources/WebDriver/Requests.swift

```diff
@@ -462,6 +462,37 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidexecute
+    public struct SessionScript: Request {
+        public var session: String
+        public var script: String
+        public var args: [String]
```

Comment:
According to spec, this could be any javascript/json value, so `Any` or `any Codable` would be a more accurate representation, if we can make this work.